### ISSUE TITLE
Generate percentages of matchs win, draw and lose

### ISSUE THREAD

**Issue by matichewer** [matichewer/BrawlStars_JavaWrapper#1]
Connect to the BrawlStars API and generate the corresponding class with the statistics of the matches (win, draw, lose). 

Maybe also what game modes have been played.

**Comment by AliMora83** [comment]
Hey @matichewer 🥳

Was this issue fixed? 🤔

**Comment by matichewer** [comment]
Hi!

I think I haven't finished this feature.
The goal of this repository was to be used in this other repo: https://github.com/matichewer/BrawlStars_TelegramBot
You can test the Telegram bot at: https://t.me/bs_stats_bot

Let me know if you have any further questions or need any additional assistance. Thank you!

**Comment by AliMora83** [comment]
It's in Spanish 🤔

<img width="378" alt="Screenshot 2023-05-25 at 22 07 36" src="https://github.com/matichewer/BrawlStars_JavaWrapper/assets/56822813/0baf3c81-5632-49b3-8da1-8ad6badabaca">

**Comment by matichewer** [comment]
Yes, sorry, I'm from Argentina. When I was developing the Telegram bot, I made it only in Spanish. 
I have an issue in the other repository regarding this: https://github.com/matichewer/BrawlStars_TelegramBot/issues/2

![Screenshot_20230525-172450-577](https://github.com/matichewer/BrawlStars_JavaWrapper/assets/48109018/fd2dd493-4de3-4e68-b41d-4818b264cef7)

**Comment by AliMora83** [comment]
No problem. Do you have a website I can help with? 🤔
